Source/SwiftLintFramework/Rules/Style/IndentationWidthRule.swift

```diff
@@ -46,14 +47,22 @@ public struct IndentationWidthRule: ConfigurationProviderRule, OptInRule {
     public init() {}
 
     // MARK: - Methods: Validation
-    public func validate(file: SwiftLintFile) -> [StyleViolation] { // swiftlint:disable:this function_body_length
+	// swiftlint:disable:next cyclomatic_complexity function_body_length
+    public func validate(file: SwiftLintFile) -> [StyleViolation] {
         var violations: [StyleViolation] = []
         var previousLineIndentations: [Indentation] = []
 
         for line in file.lines {
             // Skip line if it's a whitespace-only line
-            let indentationCharacterCount = line.content.countOfLeadingCharacters(in: CharacterSet(charactersIn: " \t"))
-            if line.content.count == indentationCharacterCount { continue }
+            let comment = "//"
```

Comment:
Should this handle other comment styles (`/*`, `///`, `/**`)?

Reply:
I added support for those too, but I don't quite like the implementation 😞 I thought that maybe `String.commentBody()` would help me, but it doesn't support `/*`: https://github.com/jpsim/SourceKitten/issues/695.